server/webhookNotifier.ts

```diff
@@ -267,11 +298,7 @@ const send = async (models, content: WebhookContent) => {
         return;
       }
       try {
-        if (process.env.NODE_ENV === 'production' || (SLACK_FEEDBACK_WEBHOOK && url === SLACK_FEEDBACK_WEBHOOK)) {
-          await request.post(url).send(webhookData);
-        } else {
-          console.log('Suppressed webhook notification to', url);
-        }
+         await request.post(url).send(webhookData);
```

Comment:
seems you removed the conditional stuff based on the environment. let's roll back this.

Reply:
Agree, and perhaps add a special case so that we can test TG webhooks when not running in production mode.